src/app/docs/definition-list/definition-list-docs.component.html

```diff
@@ -7,6 +7,34 @@
     Definition lists display label-value pairs.
   </sky-docs-demo-page-summary>
 
+  <sky-docs-demo>
+    <sky-docs-demo-control-panel
+      (selectionChange)="onDemoSelectionChange($event)"
+      >
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-checkbox
+          label="Show heading"
+          propertyName="showHeading"
+          [checked]="true"
+        >
+        </sky-docs-demo-control-panel-checkbox>
+      </sky-docs-demo-control-panel-section>
+    </sky-docs-demo-control-panel>
+    <sky-definition-list>
+      <sky-definition-list-heading *ngIf="demoSettings.showHeading">
+        List heading
```

Comment:
Should we ask UX for a more real-world example?
```suggestion
        Definition list heading
```